compiler/rustc_hir_analysis/messages.ftl

```diff
@@ -234,7 +234,9 @@ hir_analysis_inherent_ty_outside = cannot define inherent `impl` for a type outs
 
 hir_analysis_inherent_ty_outside_new = cannot define inherent `impl` for a type outside of the crate where the type is defined
     .label = impl for type defined outside of crate
-    .note = define and implement a trait or new type instead
+    .note = define and implement a trait or new type instead, for more details about orphan rules, see <https://doc.rust-lang.org/reference/items/implementations.html?highlight=orphan#orphan-rules>
```

Comment:
I'm confused by this message. 

"define and implement a trait" doesn't really make sense when I read it. We don't explain what a "new type" is; if you mean a newtype wrapper like `struct Foo(Bar);`, we could perhaps turn this into a structured suggestion.